libs/datasets/combined_datasets.py

```diff
@@ -66,29 +66,35 @@ class RegionLatestNotFound(IndexError):
 # One way of dealing with this is going from showcasing datasets dependencies
 # to showingcasing a dependency graph of transformations.
 ALL_TIMESERIES_FEATURE_DEFINITION: FeatureDataSourceMap = {
-    CommonFields.CASES: [UsaFactsDataSource, NYTimesDataset],
+    CommonFields.CASES: [CANScraperStateProviders, UsaFactsDataSource, NYTimesDataset],
     CommonFields.CONTACT_TRACERS_COUNT: [TestAndTraceData],
     CommonFields.CUMULATIVE_HOSPITALIZED: [CovidTrackingDataSource],
     CommonFields.CUMULATIVE_ICU: [CovidTrackingDataSource],
     CommonFields.CURRENT_HOSPITALIZED: [
+        CANScraperStateProviders,
+        CovidTrackingDataSource,
+        TexasHospitalizations,
+        HHSHospitalDataset,
+    ],
+    CommonFields.CURRENT_ICU: [
+        CANScraperStateProviders,
         CovidTrackingDataSource,
         TexasHospitalizations,
         HHSHospitalDataset,
     ],
-    CommonFields.CURRENT_ICU: [CovidTrackingDataSource, TexasHospitalizations, HHSHospitalDataset,],
     CommonFields.CURRENT_ICU_TOTAL: [HHSHospitalDataset],
     CommonFields.CURRENT_VENTILATED: [CovidTrackingDataSource],
-    CommonFields.DEATHS: [UsaFactsDataSource, NYTimesDataset],
+    CommonFields.DEATHS: [CANScraperStateProviders, UsaFactsDataSource, NYTimesDataset],
     CommonFields.HOSPITAL_BEDS_IN_USE_ANY: [HHSHospitalDataset],
-    CommonFields.ICU_BEDS: [HHSHospitalDataset],
+    CommonFields.ICU_BEDS: [CANScraperStateProviders, HHSHospitalDataset],
     CommonFields.NEGATIVE_TESTS: [CovidTrackingDataSource, HHSTestingDataset],
     CommonFields.POSITIVE_TESTS: [CovidTrackingDataSource, HHSTestingDataset],
     CommonFields.TOTAL_TESTS: [CovidTrackingDataSource],
     # STAFFED_BEDS isn't used right now. Disable to ease refactoring.
-    # CommonFields.STAFFED_BEDS: [CovidCountyDataDataSource],
+    # CommonFields.STAFFED_BEDS: [CANScraperStateProviders, CovidCountyDataDataSource],
     CommonFields.POSITIVE_TESTS_VIRAL: [CovidTrackingDataSource],
```

Comment:
Do you want to add positive tests viral also? ahh I think [one of the fields](https://github.com/covid-projections/covid-data-public/pull/198/files#r562250222) was incorrect in the covid-data-public PR

Reply:
Nice catch! Fixed with 225d1956